src/libraries/Pausable.sol

```diff
@@ -105,4 +160,28 @@ abstract contract Pausable is Initializable {
         PauseStorage storage $ = _getPauseStorage();
         return $._pauseFlag == PAUSED;
     }
+
+    /**
+     * @dev Transfers the ability to pause to a new account (`newPauser`).
+     */
+    function _transferPauserCapability(address newPauser) internal virtual onlyPauser {
```

Comment:
Not sure whether we want the owner or pauser to control the next pauser?

Reply:
i think it could be either. if the owner and the pauser are distinct, the owner already trusts the pauser to guarantee liveness.

Reply:
Yeah same as above, with the two roles distinct I think it makes sense to allow both to specify the next pauser

Reply:
Agreed, I think both owner and pauser should be able to specify the next pauser (in case the pauser specifies a new pauser that's malicious).